src/main/main.ts

```diff
@@ -74,6 +79,17 @@ const setWindowDimension = (isExpanded: boolean) => {
 const state: IState = fetchSettings();
 let configPath = '';
 
+try {
+  // create presets dir if it doesn't exist
+  if (!fs.existsSync(PRESETS_DIR)) {
+    fs.mkdirSync(PRESETS_DIR);
+  }
+} catch (e) {
+  console.error('Failed to make presets directory!!');
+  console.error(e);
```

Comment:
idk if throwing an error is useful here. since this is tied to the scope of the app launch, i don't think this exception does anything. Technically, all our other preset functions will gracefully throw errors should this preset folder creation fail. Its fine to leave it like this. @flyingpandaboy @selinali2010 what do yall think?

Reply:
what do you mean by exception doesn't do anything? Won't the application crash on startup

Reply:
Okay confirmed app crashes with the exception thrown.